src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,95 @@
 package core.basesyntax;
 
+import java.util.Objects;
+
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int INITIAL_CAPACITY = 16;
+    private static final float LOAD_FACTOR = 0.75f;
+    private static final int BASEINT = 2;
+    private int size;
```

Comment:
let's use more informative naming
```suggestion
    private static final int RESIZE_FACTOR = 2;
    private int size;
```